TheraLang.Web/ViewModels/UserAllViewModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TheraLang.Web.ViewModels
+{
+    public class UserAllViewModel
+    { 
+        [Required(ErrorMessage = "The email address is required")]
+        [EmailAddress(ErrorMessage = "Invalid Email Address")]
```

Comment:
Use fluent validation

Reply:
What do you mean? it's already build-in validation in .Net.

Reply:
Example:
https://github.com/ivanko-yuz/TheraLang/blob/b40946d0ad32737880481b20fe56acd7a56a2d95/TheraLang.Web/Validators/NewsValidators/NewsCreateViewModelValidator.cs#L6-L18

Reply:
Could you provide the benefits of this?

Reply:
https://stackoverflow.com/questions/6807472/fluent-validation-vs-data-annotations

And we already using fluent validation for other viewmodels

Reply:
done